storage/object/concepts.mdx

```diff
@@ -132,7 +132,7 @@ The feature uses a write-once-read-many (WORM) data protection model. This model
 
 A storage service based on the S3 protocol. It allows you to store different types of objects (documents, images, videos, etc.) and to distribute them instantly, anywhere in the world. You can upload, download and visualize stored objects.
 
-Contrary to other storage types (block devices, file-systems…), Object Storage bundles the data itself along with metadata tags and a unique identifier (instead of a file name and a file path). These unique identifiers are arranged in a flat address space which makes it easier to locate and retrieve your data across regions. This flat address space also helps with scalability.
+Contrary to other storage types (block devices, file-systems…), Object Storage bundles the data itself along with metadata [tags](#tags) and a [prefix](#prefix), rather than a file name and a file path.
```

Comment:
```suggestion
Contrary to other storage types such as block devices or file-systems, Object Storage bundles the data itself along with metadata [tags](#tags) and a [prefix](#prefix), rather than a file name and a file path.
```